apps/site/src/app/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @next/next/no-img-element */
+import Image from "next/image";
+
+import { getSponsors } from "./getSponsors";
+import { client } from "@/lib/sanity/client";
+import imageUrlBuilder from "@sanity/image-url";
+import fishingBoat from "@/assets/images/fishing-boat.png";
+
+import styles from "./Sponsors.module.scss";
+
+const builder = imageUrlBuilder(client);
+
+export default async function Sponsors() {
+	const sponsors = await getSponsors();
+
+	return (
+		<section className="container py-24 md:my-16 relative items-center flex flex-col md:p-8 w-3/4 mx-auto text-center">
+			<h2
+				className={`${styles.title} my-6 font-display sm:text-[3rem] text-[#fffce2] text-4xl text-center`}
+			>
+				Sponsors
+			</h2>
+			<p className="max-w-md mb-16 md:mb-12">
+				Interested in sponsoring IrvineHacks 2024? Check out our
+				information above to learn more about our event! For more
+				information, please email us at{" "}
+				<a href="mailto:hack@uci.edu">hack@uci.edu</a>
```

Comment:
Issue: there should be some sort of style that differentiates `hack@uci.edu` from the rest of the text (ex. an underline and hover transition).

Reply:
Nitpick: There should also be a period at the end of this sentence.